rust/exchange_rate/README.md

```diff
@@ -57,7 +57,9 @@ triggering remote HTTP calls again.
 `cargo build --target wasm32-unknown-unknown --release --p exchange_rate`
 
 ## Deploy the canister locally
+- `cargo install cargo-audit` to install vulnerability inspection tool
+- `npm i svelte` to prepare needed UI canister dependencies
```

Comment:
Generally, installing an individual package won't be necessary in a step like this, since Svelte is included in the `package.json`. If this issue came up during your process, I'd imagine that a simple `npm install` is probably all that was needed

Reply:
Got it.  I will replace `npm i svelte` to `npm install` to catch all. Thanks!

Reply:
updated